docs/deployment/openshift/falconcontainer-openshift.yaml

```diff
@@ -0,0 +1,14 @@
+﻿apiVersion: falcon.crowdstrike.com/v1alpha1
```

Comment:
This feels like a duplicate file. Can't we just use what is in config?

Reply:
I could not find the file this is duplicate to?